.gitignore

```diff
@@ -28,3 +28,5 @@ client-react/.nyc_output
 /client-react/.gh-pages-tmp
 /client-react/www/static
 /demo/.gh-pages-tmp
+
+/client-react/win
```

Comment:
I'll add it for this PR. But @estambakio-sc not forget find a way to pass env variables on MS Windows and remove this goal.